CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master (unreleased)
 
+### New features
```

Comment:
There's a section called `Changes` where we normally put changes to existing cops, although this might even qualify as a bug fix. 🙂